templates/router/connector.conf.erb

```diff
@@ -7,7 +7,7 @@ connector {
     sasl-username: <%= @sasl_username %>
 <% end -%>
 <% unless [nil, :undefined, :undef, ''].include?(@sasl_password) -%>
-    sasl-password: <%= @sasl_password %>
+    sasl-password: <%= @sasl_password_file %>
```

Comment:
Should this be `sasl-password-file` now?

Reply:
I'm not sure what you mean -- are you talking about the `unless` on L9? The file resource should only get created when `defined($sasl_password)`; also `$sasl_password_file` has a default value so I think it works this way.

Reply:
Well, if I look at https://qpid.apache.org/releases/qpid-dispatch-0.8.1/man/qdmanage.html I see `sasl-password-file`. If I look at https://qpid.apache.org/releases/qpid-dispatch-1.14.0/man/qdrouterd.conf.html it appears you would need to do `file:<%= @sasl_password_file %>`.

Reply:
Ah, yeah. I was looking at https://qpid.apache.org/releases/qpid-dispatch-1.14.0/man/qdrouterd.conf.html but missed the `file:`. Thanks!